CHANGES.md

```diff
@@ -74,6 +75,246 @@
 * [Version 1.0.1](#version-101)
 * [Version 1.0](#version-10)
 
+## Version 3.3.0 (September 30, 2024)
+
+Minor release two significant changes:
+
+- The C++ API now uses the OpenEXRCore library underneath.
+
```

Comment:
Maybe we can summarize why this is important? Is there a perf benefit?

Reply:
Kimball said he's got notes, I left that section for him